### ISSUE TITLE
Data Query Debug section closes when trying to highlight

### ISSUE THREAD

**Issue by netniV** [Cacti/cacti#1168]
If you are editing a device and run the Verbose Query option, you get the standard Data Query Debug Information layout which lists the indexes and then headings for each of the fields.  Clicking a heading will automatically expand the field section to give you the values.

However, if you wish to copy and paste those values, you will be unable to because the section automatically closes again.  So, we need a way to either copy a section (link on the right of the header?) or prevent the section automatically closing when single clicking on the data (maybe allow double clicking if possible?)

**Comment by netniV** [comment]
I'm am liking the copy to clipboard idea a bit more now I think of it.  Using something like:
https://www.w3schools.com/howto/tryit.asp?filename=tryhow_js_copy_clipboard

```
<!DOCTYPE html>
<html>
<body>

<p>Click on the button to copy the text from the text field. Try to paste the text (e.g. ctrl+v) afterwards in a different window, to see the effect.</p>

<input type="text" value="Hello World" id="myInput">
<button onclick="myFunction()">Copy text</button>

<p>The document.execCommand() method is not supported in IE9 and earlier.</p>

<script>
function myFunction() {
  var copyText = document.getElementById("myInput");
  copyText.select();
  document.execCommand("Copy");
  alert("Copied the text: " + copyText.value);
}
</script>

</body>
</html>
```

**Comment by cigamit** [comment]
I agree with that approach, plus the icon for dispatching should not be the plus sign.

**Comment by netniV** [comment]
I've done a preliminary update for this in my repo but it's not in a working state for the following reasons:

- Clipboard doesn't seem to be getting updated despite execCommand being run.
- CSS needs to be defined to put the copy on the right side
- Section header now collapses presumably because an existing function is hiding the classes it doesn't recognise.

**Comment by netniV** [comment]
Updated now to fix two problems, but not happy with the layout:

- Table is now copied as HTML so can be pasted with formatting
- Section Header no longer collapses but copy command is now above the desired section
- CSS still needs to be applied

Also noticed that there is a "+" which collapses the debug section.  However, I suspect that should be changed to a "-" which makes more sense.

**Comment by netniV** [comment]
I have now created a pull request but I think it needs a little bit more love to make it 100%.  Maybe you could take a look and see what you think?

**Comment by cigamit** [comment]
Will review shortly.

**Comment by cigamit** [comment]
This is resolved in 1.1.31.
